src/components/atoms/InputText/InputText.tsx

```diff
@@ -44,7 +44,7 @@ const Input = styled.input<InputThemeProps>`
   padding: ${({ theme }: InputThemeProps) => theme.input.padding};
   padding-left: ${({ startIcon, theme }: InputThemeProps) =>
     startIcon && theme.input.startIcon ? theme.input.startIconPaddingLeft : null};
-  padding-right: ${({ endIcon }: InputThemeProps) => !!endIcon && '60px'};
+  padding-right: ${({ endIcon, theme }: InputThemeProps) => endIcon && theme.input.startIconPaddingRight};
```

Comment:
Double checking:  does it mean to be `startIconPaddingRight` as it passed in endIcon

Reply:
good spot! youre right